scripts/build_erddap_catalog.py

```diff
@@ -570,16 +570,19 @@ def build_erddap_catalog_chunk(data_root, deployment):
 
         qartod_var_type = check_for_qartod_vars(ds)
```

Comment:
this function only lists the variables that start with qartod. 
recently pushed deloyed-mode datasets have slightly modified names that do not start with qartod: https://gliders.ioos.us/erddap/tabledap/crate-20200130T0820-delayed.html